tests/components/point/test_config_flow.py

```diff
@@ -33,11 +33,15 @@ def mock_pypoint(is_authorized):  # pylint: disable=redefined-outer-name
     with patch(
         "homeassistant.components.point.config_flow.PointSession"
     ) as PointSession:
-        PointSession.return_value.get_access_token.return_value = {
-            "access_token": "boo"
-        }
+        PointSession.return_value.get_access_token = (
+            AsyncMock(  # pylint:disable=protected-access
```

Comment:
Why do we need to disable protected access?

Reply:
I think that it is your suggestion..
Check line 17 in the same file.

Ah I see it. 
```suggestion
            AsyncMock(
```